packages/nextjs/app/builders/page.tsx

```diff
@@ -0,0 +1,100 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { BuilderPicture, MentorPicture } from "./_components/index";
+import type { NextPage } from "next";
+import { useScaffoldEventHistory } from "~~/hooks/scaffold-eth";
+import { Builder, Mentor } from "~~/types/builders";
+
+const getRandomGender = () => {
+  return Math.random() < 0.5 ? "men" : "women";
+};
+
+const getRandomUserImage = () => {
+  const randomInt = Math.floor(Math.random() * 50);
+  const randomGender = getRandomGender();
+
+  return `https://randomuser.me/api/portraits/${randomGender}/${randomInt}.jpg`;
+};
+
+const getBuilderFromEvent = (event: any): Builder => ({
+  image: getRandomUserImage(),
```

Comment:
I wonder what else we can do for profile pictures 🤔 

Your design does a great job emphasizing the rounded portraits, so it seems silly to have random placeholders in the final version.
- Is it possible to map builders to their GitHub profiles and fetch those avatars, just like you do for mentors? 
- If not, there are also BlockieAvatars ([SE-2 docs](https://docs.scaffoldeth.io/components/BlockieAvatar)) you can modify.
- [Here's](https://github.com/BuidlGuidl/buidlguidl.com/blob/1cbcaa5a09d7bf276a550af53663cdfbab73f9d0/packages/nextjs/components/scaffold-eth/BlockieAvatar.tsx#L5) where they are used in the BuidlGuidl app!
<img width="1264" alt="Screenshot 2024-09-19 at 2 52 20 PM" src="https://github.com/user-attachments/assets/9c183cb2-8775-42a3-85da-b6a9b940a133">

Reply:
I have tried to obtain the GitHubs of the builders automatically, but I haven't found a way. For this, I suppose we would need to manually keep a record of all users and their GitHubs. I have therefore decided to use BlockieAvatars for the image.

Maybe somebody can help with this and find a way to get github profiles automatically.

Reply:
we could use ENS avatars. let me try something out...